.github/workflows/workflow.yml

```diff
@@ -31,9 +31,9 @@ jobs:
         chmod 600 spec/fixtures/.netrc
     - name: Install dependencies
       run: |
-        gem install bundler
-        gem update bundler
-        bundle install --without development --jobs 4 --retry 3
+        gem install bundler --no-document
+        bundle config set without development
```

Comment:
Fix the following Bundler warning:

> [DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set without 'development'`, and stop using this flag